plugins/httpd-client.ts

```diff
@@ -0,0 +1,20 @@
+export default defineNuxtPlugin(() => {
+  const {
+    public: { openFetch: clients },
+  } = useRuntimeConfig()
```

Comment:
```suggestion
  const { public: { openFetch: clients } } = useRuntimeConfig()
```

Nit, but let's do this if the linter allows. Simpler to read IMO.

Reply:
Unfortunately it doesn't.